### PR TITLE
[Chore] Setup linting tools

### PR DESCRIPTION
**Pull request by tyrro**
## What happened 👀

Closes https://github.com/tyrro/react-survey/issues/4

Following linting tools have been setup in this PR:
- Eslint: Eslint is already [integrated](https://nextjs.org/docs/basic-features/eslint) into nextjs, extended to `@nimblehq/eslint-config-nimble`
- Prettier: Also [integrated](https://nextjs.org/docs/basic-features/eslint#prettier), used a custom config file
- Stylelint: Added a custom config file
- lint-staged: Runs [linters](https://github.com/okonet/lint-staged) against staged files

## Insight 📝

N/A
 
## Proof Of Work 📹

Configuration files are added

### CODE REVIEW

**Comment by malparty** [comment]
The lint config looks good to me :) 

Should we do the following too? (feel free to challenge as usual ;-) )
- Parse existing `.js` files (currently they do not apply the linting rules)
- Add an `scss` index file to confirm the stylelint command is working?

**Comment by tyrro** [comment]
> * Parse existing `.js` files (currently they do not apply the linting rules)
>
that's a good idea. I actually took this opportunity to include [lint-staged](https://github.com/okonet/lint-staged) as pre-commit hooks to run eslint, prettier and stylelint!

> * Add an `scss` index file to confirm the stylelint command is working?
>
sure, I renamed the existing `*.css` to `*.scss` and ran the stylelint command!

File: prettier.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  printWidth: 100,
```
[the remark this answers is not in the document]
Comment (tyrro) [reply prettier.config.js:2]: Thanks @carryall , updated!

**Review by carryall — APPROVED** [review]

**Review by malparty — APPROVED** [review]
Neat! LGTM 🚀

**Pull request merged**